### DISCUSSION TITLE
Web Browser?

### DISCUSSION THREAD

**Discussion by dexxplay** [VendelinSlezak/BleskOS#84] [Q&A] [answered]
is there plans to get the web browser to fully view web pages and download/upload files from the web?
i do know that this is not a easy task but a working web browser would be a large step!

**Comment by VendelinSlezak** [comment, marked as answer]
Yes, there is. Now there is very simple text browser, but I am planning much more robust version that will be able to show images and will have partial support of CSS. I already even created some design of it. However it will take a while until I will have time to continue this project.

**Comment by defdefred** [comment]
How about a Gemini Browser? Gemini is much lighter and suitable for old hardware :-)
Gemini server could be cool too but it is another challenge making an app capable of serving multiple requests in the same time!

**Reply by VendelinSlezak** [reply]
Is not Gemini only for pages that are written for Gemini?

**Reply by defdefred** [reply]
Yes gemini is a light "web" protocol.
